backend/lcfs/web/api/user/views.py

```diff
@@ -44,100 +45,164 @@ async def get_current_user(request: Request, response: Response = None) -> UserB
+                data={},
+                error={"message": "No users found"},
+            )
+        return EntityResponse(
+            status=status.HTTP_200_OK,
+            data=users,
+            total=len(users),
+            error={},
+            success=True,
+            message="Success",
+        )
     except Exception as e:
         response.status_code = status.HTTP_500_INTERNAL_SERVER_ERROR
         logger.error("Error getting users", str(e))
-        return EntityResponse(status=status.HTTP_500_INTERNAL_SERVER_ERROR,
-                              message="Failed", success=False, data={},
-                              error={"message": f"Technical error: {e}"})
-
-
-@router.get("/{user_id}", response_model=EntityResponse, status_code=status.HTTP_200_OK)
-async def get_user_by_id(user_id: int, response: Response = None) -> EntityResponse:
-    try:
-        user = await user_repo.get_user(user_id=user_id)
-        if user is None:
-            err_msg = f"User {user_id} not found"
-            logger.error(err_msg)
-            response.status_code = status.HTTP_404_NOT_FOUND
-            return EntityResponse(status=status.HTTP_404_NOT_FOUND,
-                                  message="Not Found", success=False, data={},
-                                  error={"message": err_msg})
-        return EntityResponse(status=status.HTTP_200_OK, data=user,
-                              total=1, error={}, total_pages=1, limit=1,
-                              success=True, message="Success")
-    except Exception as e:
-        response.status_code = status.HTTP_500_INTERNAL_SERVER_ERROR
-        logger.error(f"Error finding user {user_id}", str(e))
-        return EntityResponse(status=status.HTTP_500_INTERNAL_SERVER_ERROR,
-                              message="Failed", success=False, data={},
-                              error={"message": f"Technical error: {e}"})
-
+        return EntityResponse(
+            status=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            message="Failed",
+            success=False,
+            data={},
+            error={"message": f"Technical error: {e.args[0]}"},
+        )
 
-@router.post("/create", response_model=EntityResponse, status_code=status.HTTP_201_CREATED)
-async def create_user(user_data: UserCreate, db: AsyncSession = Depends(get_async_db)) -> EntityResponse:
-    try:
-        user_repo = UserRepository(db)
-        created_user = await user_repo.create_user(user_data)
-        return EntityResponse(status=status.HTTP_201_CREATED, data=created_user, error={}, success=True, message="User created successfully")
-    except Exception as e:
-        raise HTTPException(status_code=500, detail=f"Failed to create user: {str(e)}")
+# TODO: yet to redefine and test endpoints for below.
+# @router.get("/{user_id}", response_model=EntityResponse, status_code=status.HTTP_200_OK)
+# async def get_user_by_id(user_id: int, response: Response = None) -> EntityResponse:
```

Comment:
Are these just not working currently?

Reply:
they were failing so just commented. Will make further changes as Admin page is refined along with the organization ticket that I've picked up currently.